baseapp/grpcserver.go

```diff
@@ -10,10 +10,14 @@ import (
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
 
-	servergrpc "github.com/cosmos/cosmos-sdk/server/grpc"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const (
+	// GRPCBlockHeightHeader is the gRPC header for block height.
+	GRPCBlockHeightHeader = "x-cosmos-block-height"
```

Comment:
Good idea, but I'd define this somewhere else. I don't think it's advisable to have to import baseapp just to get this const.

Reply:
Because of an import cycle:
```
package github.com/cosmos/cosmos-sdk/client
	imports github.com/cosmos/cosmos-sdk/server/grpc
	imports github.com/cosmos/cosmos-sdk/server/types
	imports github.com/cosmos/cosmos-sdk/server/api
	imports github.com/cosmos/cosmos-sdk/client
```

I'll put it in `"github.com/cosmos/cosmos-sdk/types/grpc"` then